rfc/5/responses/1/index.md

```diff
@@ -0,0 +1,505 @@
+    The output of these `coordinateTransformations` MUST be the default coordinate system, which is the last entry in the list of coordinate systems.
+- **Inside `multiscales > coordinateTransformations`**: One MAY store additional transformations here. 
+    The `input` to these transformations MUST be the default coordinate system and the `output` can be another coordinate system defined under `multiscales > coordinateSystems`.
+- **Parent-level `coordinateTransformations`**: Transformations between two or more images MUST be stored in the parent-level `coordinateTransformations` group.
+    The `input` to these transformations MUST be paths to the respective images. The `output` can be a path to an image or the name of a coordinate system.
+    If both path and name exist, the path takes precedence.
+    The authoritiative coordinate system under `path` is the *first* coordinate system in the list.
+
+This separation of transformations (inside `multiscales > datasets`, under `multiscales > coordinateTransformations` and under parent-level `coordinateTransformations`) 
+provides flexbility for different usecases while still maintaining a level of rigidity for implementations.
+
+````{admonition} Example
+
+Consider the following example for the use of all possible places to store `coordinateTransformations`.
+It comes from the SCAPE microscopy context, where lightsheet stacks are acquired under a skew angle,
+and need to be *deskewed* with an affine transformation. The acquired stack also comes with a set of relevant microscope motor coordinates,
+which place the object in world coordinates. 
+One may wish to attach the affine transformation to the multiscales itself, without having to read the parent-level zarr group that defines the world coordinate system.
+
+The `multiscales` metadata contains this:
+```json
+{
+    "multiscales": [{
+        "version": "0.5-dev",
+        "name": "example",
+        "coordinateSystems" : [
+            {
+            "name" : "deskewed",
+            "axes": [
+                {"name": "z", "type": "space", "unit": "micrometer"},
+                {"name": "y", "type": "space", "unit": "micrometer"},
+                {"name": "x", "type": "space", "unit": "micrometer"}
+                ]
+            },
+            {
+            "name" : "physical",
+            "axes": [
+                {"name": "z", "type": "space", "unit": "micrometer"},
+                {"name": "y", "type": "space", "unit": "micrometer"},
+                {"name": "x", "type": "space", "unit": "micrometer"}
+                ]
+            }
+        ],
+        "datasets": [
+            {
+                "path": "0",
+                // the transformation of other arrays are defined relative to this, the highest resolution, array
+                "coordinateTransformations": [{
+                    "type": "identity",
+                    "input": "/0",
```

Comment:
I think we say that the `input` should match the `path` for each dataset?

Reply:
Fixed in 0fdc5ddc9e8422a8d58d728e54a1485f88eaf072 👍